package.json

```diff
@@ -12,6 +12,10 @@
   ],
   "scripts": {
     "start": "cd infrastructure/develop && ./compose-build.sh && ./run-servers.sh dev-build",
+    "start:v2": "concurrently \"yarn dev:server:v2\" \"yarn dev:app\" \"yarn dev:proxy\"",
+    "dev:server:v2": "cd api-v2 && yarn dev",
+    "dev:app": "cd frontend && yarn start",
```

Comment:
later on when we switch to yarn workspaces, can do `yarn workspace <workspace> <cmd>` see [here](https://classic.yarnpkg.com/en/docs/cli/workspace)

Reply:
yeap already have the change in the workspace branch!